output/cloud/expv2/output.go

```diff
@@ -213,6 +286,7 @@ func (o *Output) handleFlushError(err error) {
 
 	o.logger.WithError(err).Warn("Interrupt sending metrics to cloud due to an error")
 
+	// TODO(lukasz): This is not thread-safe
```

Comment:
Master rebase required. Setting mostly as a reminder.

Reply:
Done. Also, I removed the TODO now.